src/client/components/Balace.js

```diff
@@ -1,19 +1,19 @@
 import React from 'react'
+import axios from 'axios'
 
 export default class Balance extends React.Component {
   constructor(props) {
     super(props)
+    
     this.state = {
       balance: 0
     }
   }
   
   componentDidMount() {
-    fetch('/api/me', {
-      method: 'GET'
-    })
-      .then(response =>{
-        return response.json()
+    axios.get('/api/me')
+      .then(response => {
+        return response.data
```

Comment:
zamiast 2x then(), mozesz zrobic:
`.then(response => this.setState(response.data)`

Reply:
Poprawione